lib/fizzy/parser.cpp

```diff
@@ -275,14 +276,41 @@ Module parse(bytes_view input)
         }
 
         if (it != expected_end_pos)
+        {
             throw parser_error{"incorrect section " + std::to_string(static_cast<int>(id)) +
                                " size, difference: " + std::to_string(it - expected_end_pos)};
+        }
+    }
+
+    if (module.tablesec.size() > 1)
```

Comment:
Bringing these two existing checks out of the loop, too.

Reply:
I think they were in there because a strict order is expected and there would be safe to check and abort early. However we do not check the ordering (yet).

Reply:
Would an early abort be beneficial btw?

Reply:
Might be benificial, but the way it was it's for the price of doing the checks after each section.